example/relational/README.md

```diff
@@ -178,7 +180,14 @@ Composite sort key supports hierarchical one-to-many relations. Additional ortho
 * As a reader I want to list all articles written by the author ...
 * As a reader I want to look up all articles for a given category in chronological order ...
 
-The first access pattern is addressed by composite sort key `author ⟼ article`, the second one requires another `category ⟼ year` key. One approach is an explicit projection of data but secondary indexes are easy. DynamoDB implicitly copies data from the main table into the secondary index using another pair of attributes to establish identity. Therefore a new access dimension is unlocked. Eventual consistency is only the aspect to consider while using indexes. The local secondary indexes provide strong consistency but general advice to favour global indexes.
+The first access pattern is addressed by composite sort key `author ⟼ article`,
+=> this is a bit disturbing to me ^^' isn't composite partition key?, is it the same? 
```

Comment:
Thank you for emphasising it, Looks like an editorial changes are required but let'e clarify this.
The article ID is defined by the tuple (see the example class definition https://github.com/fogfish/dynamo/blob/main/example/relational/types.go#L67)
```
( author:neumann, article:theory_of_automata )
```
at first glance, it is a normal tuple. Where is the composed sort key as it is defined by AWS documentation>
> Composite sort keys let you define hierarchical (one-to-many) relationships in your data that you can query at any level of the hierarchy. e.g. [country]#[region]#[state]#[county]#[city]#[neighborhood]

but the article strongly recommends to use IRI type to define hierarchy. Here it is schema (`article`) + id (`theory_of_automata`) defines composite sort key.

This application can introduces a new "relations" out of the box
```

( author:neumann, notes:theory_of_automata )
( author:neumann, article:wired_magazine/theory_of_automata )
...
```